apollo-ios/Sources/Apollo/ApolloStore.swift

```diff
@@ -232,6 +232,7 @@ public class ApolloStore {
       return try executor.execute(
         selectionSet: type,
         on: object,
+        expecting: .all,
```

Comment:
`all` seems correct here based on what we spoke about yesterday @AnthonyMDev, but please double-check. It'll be reviewed when the cache work is done too.